### ISSUE TITLE
No 240p FLV, 480p FLV, 1080p MP4 

### ISSUE THREAD

**Issue by bo3rnd** [hoyois/clicktoplugin#50]
Hi,

I just saw that there was an update for clicktoflash on 06.11.2013 with these notes:
"Updated the YouTube killer with support for 480p FLV, 1080p MP4, and 1440p MP4 videos"

But I can only select 360p MP4 and 720p MP4.
What happened with the other formats?

thanks

**Comment by saboch** [comment]
@hoyois there is an open source HTML5 player that seems fairly active and has plugin support for dash.js and dash.as, this all isn't really my forte but the results looked promising! http://blog.videojs.com/post/92536319027/dash-everywhere-ish-hack-project#disqus_thread

**Comment by asoksevil** [comment]
Any new updates? AFAIK 360p and 720p are the only options available.

**Comment by hoyois** [comment]
Nope, nothing new. It's probably best not to expect anything. At least we can be grateful that the 720p video is still there, now that all main browsers support MSE.

**Comment by luizfonseca** [comment]
Guys, what about this? https://github.com/Dash-Industry-Forum/dash.js/tree/v1.2.0/contrib/videojs

**Comment by bretello** [comment]
Any news on this?
